Buildings/Examples/VAVReheat/Guideline36_DXSystems.mo

```diff
@@ -0,0 +1,159 @@
+within Buildings.Examples.VAVReheat;
+model Guideline36_DXSystems
```

Comment:
Run the model gives warning as:
<img width="1003" alt="Screenshot 2024-05-13 at 1 32 27 PM" src="https://github.com/lbl-srg/modelica-buildings/assets/23491921/8032eecd-40c2-4cf2-9338-e8d5108b2837">

Also, it takes more than 10 minutes to finish the simulation, while for `Guideline36.mo` it takes 18 seconds.